geode-lucene/src/main/java/com/gemstone/gemfire/cache/lucene/internal/LuceneIndexForPartitionedRegion.java

```diff
@@ -77,13 +77,13 @@ public void initialize() {
       final String fileRegionName = createFileRegionName();
       PartitionAttributes partitionAttributes = dataRegion.getPartitionAttributes();
       if (!fileRegionExists(fileRegionName)) {
-        fileRegion = createFileRegion(regionShortCut, fileRegionName, partitionAttributes);
+        fileRegion = createFileRegion(regionShortCut, fileRegionName, partitionAttributes, ra);
```

Comment:
Its a nitpicky stuff but can we rename ra as regionAttributes. Just to keep things uniform... all other variables have full explanation names like regionShortcut, fileRegionName but just "ra" is the extra short one.